AutoStructureFilesForDFplayer.py

```diff
@@ -2,6 +2,8 @@
 import sys
 import os.path
 import shutil
+# urllib should do what you need in replaceSpecialCharactersInPlaylist

```

Comment:
re-formulate comment, e.g. "use urllib to escape special (UTF8) characters in filenames"